addon/components/base/bs-form.js

```diff
@@ -75,13 +75,13 @@ import layout from 'ember-bootstrap/templates/components/bs-form';
   A `isSubmitting` property is yielded, which is `true` after submit has been triggered and before the Promise returned
   by `onSubmit` is fulfilled. It could be used to disable form's submit button and showing a loading spinner for example:
 
-  ```
-  {{#bs-form onSubmit=(action 'save') as |form|}}
-    {{#bs-button buttonType='submit' disabled=form.isSubmitting}}
+  ```hbs
+  <BsForm @onSubmit={{action "save"}} as |Form|>
+    <BsButton @buttonType="submit" @disabled={{form.isSubmitting}} >
```

Comment:
There shouldn't be space between `}}` and `>`, should it?

Reply:
I'll just remove them for now